test_util/test_installer_ccm.py

```diff
@@ -381,8 +394,17 @@ def make_runner(host_list):
     print("VPC hosts: {}".format(host_list))
     # use first node as bootstrap node, second node as master, all others as agents
     registry_host = local_ip[host_list[0]]
-    master_list = [local_ip[_] for _ in host_list[1:2]]
-    agent_list = [local_ip[_] for _ in host_list[2:]]
+    master_list = []
+    agent_list = []
+    print('Test/registry host public/private IP: ' + host_list[0] + '/' + registry_host)
+    for i in range(1, (options.num_masters+1)):
```

Comment:
The goal was to print out the hosts in such a way that one could easily parse the local and public IP for a single node